hard/fakeram/bsg_mem_1rw_sync_macros.vh

```diff
@@ -0,0 +1,19 @@
+
+`ifndef BSG_MEM_1RW_SYNC_MACROS
+`define BSG_MEM_1RW_SYNC_MACROS
+
+`define bsg_mem_1rw_sync_1sram_macro(words,bits,tag) \
+  if (els_p == words && width_p == bits)             \
+    begin: macro                                     \
+       fakeram_d``words``_w``bits`` mem              \
+         (.clk(i_clk)                                \
+         ,.ce_in(clk_en_lo)                          \
+         ,.we_in(w_i)                                \
+         ,.wd_in(data_i)                             \
+         ,.w_mask_in(-1)                             \

```

Comment:
'1

Reply:
?

Reply:
-1 is inherently a 32b integer, which will cause a width mismatch/synthesizer error. 

'1 is the SystemVerilog shorthand for "as many 1s as needed to match this bus width"

Reply:
Oh I didn't know that. Nice!

Reply:
Correspondingly '0 is very useful :)